qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,311 @@
+from .ATS import AlazarTech_ATS
+from .utils import TraceParameter
+from qcodes.utils import validators
+
+
+class AlazarTech_ATS9440(AlazarTech_ATS):
+    """
+    This class is the driver for the ATS9440 board
+    it inherits from the ATS base class
+    """
+    samples_divisor = 256 #32
+
+    def __init__(self, name, **kwargs):
+        dll_path = 'C:\\WINDOWS\\System32\\ATSApi.dll'
+        super().__init__(name, dll_path=dll_path, **kwargs)
+
+        # add parameters
+
+        # ----- Parameters for the configuration of the board -----
+        self.add_parameter(name='clock_source',
+                           parameter_class=TraceParameter,
+                           get_cmd=None,
+                           label='Clock Source',
+                           unit=None,
+                           initial_value='INTERNAL_CLOCK',
+                           val_mapping={'INTERNAL_CLOCK': 1,
+                                               'FAST_EXTERNAL_CLOCK': 2,
+                                               'SLOW_EXTERNAL_CLOCK': 4,
+                                               'EXTERNAL_CLOCK_10MHz_REF': 7})
+        self.add_parameter(name='external_sample_rate',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Sample Rate',
+                           unit='S/s',
+                           vals=validators.MultiType(validators.Ints(1000000, 125000000),
+                                                     validators.Enum('UNDEFINED')),
+                           initial_value='UNDEFINED')
```

Comment:
The card supports slow external clock for the range below 1 MHz.
https://www.alazartech.com/Support/Download%20Files/ATS9440%20User%20Manual_V1_0.pdf (page 38).
I think this future should be implemented or specified as TODO to be done in the future.